components/tools/OmeroPy/test/integration/clitest/test_duplicate.py

```diff
@@ -39,6 +39,28 @@ def duplicate(self, capfd):
         self.cli.invoke(self.args, strict=True)
         return capfd.readouterr()[0]
 
+    def get_dataset(self, iid):
+        """Retrieve all the parent datasets linked to the image"""
+
+        params = omero.sys.ParametersI()
+        params.addId(iid)
+        query = "select d from Dataset as d"
```

Comment:
You can use the following Python syntax to neaten this up a bit:
```
query = ("select d from Dataset as d"
              " where exists ("
              "..." )
```
ie if you use brackets you can break strings over several lines.